SofaKernel/framework/sofa/helper/vector.cpp

```diff
@@ -51,6 +52,62 @@ void SOFA_HELPER_API vector_access_failure(const void* vec, unsigned size, unsig
     assert(i < size);
 }
 
-} // namespace helper
+/// Convert the string 's' into an unsigned int. The error are reported in msg & numErrors
+/// is incremented.
+int SOFA_HELPER_API getInteger(const std::string& s, std::stringstream& msg, unsigned int& numErrors)
+{
+    const char* attrstr=s.c_str();
```

Comment:
@maxime-tournier your help is welcome, 

I would prefer this PR to be merged quickly so that we can start working on improving/refactoring/accelerating/make it more c++ for both the 'vector' & 'set' implementation in #278 (that is dedicated to this task) because the code seems very duplicated. 

The benchmark is in vector_test... There is only one benchmark (to detect regression) so you have to recompile the different versions.